test/integration/proxy_test.go

```diff
@@ -35,7 +35,9 @@ var _ = Describe("", Serial, Ordered, Label("openshift-preset", "goproxy"), func
 		// Start goproxy
 
 		It("configure CRC", func() {
-			Expect(RunCRCExpectSuccess("config", "set", "network-mode", networkMode), ContainSubstring("Network mode"))
+			if runtime.GOOS == "linux" {
+				Expect(RunCRCExpectSuccess("config", "set", "network-mode", networkMode), ContainSubstring("Network mode"))
+			}
```

Comment:
I'd argue that it's not very useful to test a non-default network mode, as this is not what most users will experience. Are there some issues if we run with the default network mode on linux?

Reply:
@jsliacan WDYS?

Reply:
The reason why it was this way was mostly convenience - at the time when I wrote that test, you could set network mode on all platforms, so no extra code needed for linux. Also, linux system mode is subject to the e2e proxy test (which is only for linux). So this seemed to cover another (albeit less useful) case of user mode on linux.

That said, I see how it's a non-default option and thus might not be worth testing at this point. If you say that that's the case I'm happy to parametrize the proxy address by platform and use system mode on linux.

Reply:
>  If you say that that's the case I'm happy to parametrize the proxy address by platform and use system mode on linux.

Ah, if this makes the proxy test/code simpler, this can be a reason for using `user` mode networking on linux. Let's keep this for now to try to fix integration tests.

Reply:
With system networking, it's https://github.com/crc-org/crc/blob/85cc7183e4daf12b148b594a444e1d50546910e3/test/extended/util/proxy.go#L68 which would need to be changed to something reachable by the VM?

Reply:
I'm surprised this works with user-mode networking though if you don't' set `* host-network-access                        Allow TCP/IP connections from the CRC VM to services running on the host (true/false, default: false)`

Reply:
NB: should have looked at your recent changes before commenting ^^